api/azure/flavors.go

```diff
@@ -0,0 +1,141 @@
+package azure
+
+import (
+	"github.com/elastisys/ck8s/api"
+)
+
+const (
+	FlavorDevelopment api.ClusterFlavor = "dev"
+	FlavorProduction  api.ClusterFlavor = "prod"
+)
+
+// Common sizes:
+// More details here: https://docs.microsoft.com/en-us/azure/virtual-machines/dv2-dsv2-series
+// +-----------------+------+-----------+------------+----------------+----------+
+// | Size            | vCPU | RAM (GiB) | Disk (GiB) | Max data disks | Max NICs |
+// +-----------------+------+-----------+------------+----------------+----------+
+// | Standard_D2_v3  | 2    | 8         | 50         | 4              | 2        |
+// | Standard_D4_v3  | 4    | 16        | 100        | 8              | 2        |
+// | Standard_D8_v3  | 8    | 32        | 200        | 16             | 4        |
+// | Standard_D16_v3 | 16   | 64        | 400        | 32             | 8        |
+// | Standard_D32_v3 | 32   | 128       | 800        | 32             | 8        |
+// | Standard_D48_v3 | 48   | 192       | 1200       | 32             | 8        |
+// | Standard_D64_v3 | 64   | 256       | 1600       | 32             | 8        |
+// +-----------------+------+-----------+------------+----------------+----------+
+
+func Default(clusterType api.ClusterType, clusterName string) *Cluster {
+	return &Cluster{
+		config: AzureConfig{
+			BaseConfig: *api.DefaultBaseConfig(
+				clusterType,
+				api.Azure,
+				clusterName,
+			),
+			// TODO change this to azure storage
+			S3RegionAddress: "sos-ch-gva-2.exo.io",
```

Comment:
We should probably fix this before merging, no?

Reply:
It would take a bit too much work I think. Azure does not have any s3 compatible storage. So we would either have to deploy our own s3 service (e.g. minio) or see if all of our apps can use azures api instead of s3.

So it feels better to just get this merged for now.

Reply:
Ok, I think this needs to be highlighted clearly so that no one get's the impression that we can run on this in production tomorrow. Which sometimes is the case... :smile: 

Please create follow-up issues for what is left to be done too.